Templating/VOKTemplateModel.m

```diff
@@ -25,7 +25,7 @@ @interface VOKTemplateModel ()
 static NSString *const KitNameIOS = @"UIKit";
 static NSString *const KitNameMac = @"AppKit";
 
-static NSString *const FrameworkPrefix = @"ac_";
+static NSString *const MethodNamePrefix = @"ac";
```

Comment:
I played around with that a bit. However, I saw some strange behavior with GRMustache. It seemed to want to "trim" the trailing `_`. I may have been doing something wrong though.